elide-async/src/main/java/com/yahoo/elide/async/service/storageengine/RedisResultStorageEngine.java

```diff
@@ -29,7 +28,7 @@
 @Slf4j
 @Getter
 public class RedisResultStorageEngine implements ResultStorageEngine {
-    @Setter private JedisPool jedisPool;
+    @Setter private UnifiedJedis jedis;
```

Comment:
Switching to UnifiedJedis so we can switch between JedisCluster and JedisPooled according the environment.

https://github.com/redis/jedis/blob/master/README.md
https://github.com/redis/jedis/releases/tag/v4.0.0